src/megaapi_impl.cpp

```diff
@@ -20543,9 +20558,11 @@ void MegaApiImpl::sendPendingRequests()
             if (!megaTransfer->isStreamingTransfer())
             {
                 Transfer *transfer = megaTransfer->getTransfer();
-                if (!transfer)
+                if (!transfer) //folder transfer
```

Comment:
Note for reviewer, there might be a better way to identify a folder transfer

Reply:
What about `MegaTransfer::isFolderTransfer()`??